### PR TITLE
Updated find_prefix function, find_prefix tests, and deleted test_pyeasytries.

### PR DESCRIPTION
**Pull request by rainbowxyt0305**

### CODE REVIEW

File: pyeasytries/Trie.py [hunk 1]
```diff
@@ -69,7 +99,7 @@ def add(self, word_to_add):
         >>> trie.add("firetruck")
         TRUE 
         """
-
```
Comment (Andrew-Tan) [inline pyeasytries/Trie.py:92]: Could you undo this change? This is part of a line for `add()` so I worried this line might create merge conflict later when we are trying to merge Jordan's PR

**Review by Andrew-Tan — CHANGES_REQUESTED** [review]
Great work! Just a minor suggestion regarding undoing a change near function `add()` to prevent merge conflict with Jordan's PR.

File: pyeasytries/Trie.py (hunk 1 above)
Comment (rainbowxyt0305) [reply pyeasytries/Trie.py:92]: Sure on it!

Comment (rainbowxyt0305) [reply pyeasytries/Trie.py:92]: Undone! Thanks Kaicheng!

**Comment by rainbowxyt0305** [comment]
Updated RP with changes you requested @Andrew-Tan !

**Review by Andrew-Tan — APPROVED** [review]
LGTM

**Pull request merged**